sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -48,6 +49,31 @@
 _ENABLE_GOOGLE_CLOUD_PROFILER = 'enable_google_cloud_profiler'
 
 
+def _import_beam_plugins(plugins):
+  for plugin in plugins:
+    try:
+      importlib.import_module(plugin)
+      _LOGGER.info('Imported beam-plugin %s', plugin)
+    except ImportError:
+      try:
+        _LOGGER.debug(
+          (
+            "Looks like %s is not a module. "
+            "Trying to import it assuming it's a class"
+          ),
+          plugin
+        )
+        module, _ = plugin.rsplit('.', 1)
+        importlib.import_module(module)
+        _LOGGER.info('Imported %s for beam-plugin %s', module, plugin)
+      except ImportError:
+        _LOGGER.warning(
```

Comment:
Should this instead be a fatal error?

Reply:
I feel if a plugin isn't imported we should continue with the pipeline execution, would love to hear your rationale about why would we want to make this fatal?